docs/dev/compliance.md

```diff
@@ -0,0 +1,120 @@
+# About `inspec compliance`
+
+## Purpose
+
+The `compliance` set of subcommands handle user-initiated communication with Chef Automate. The commands are provided so that a user can interact with an Automate installation.
+
+`inspec compliance` is somewhat analogous to `knife` in that it can be used to upload, download and manage profiles for distribution to other clients.
+
+When Automate initiates scans, this subcommand is not used.
+
+## Operational Notes
+
+### Obtaining an a test Automate server
+
+You'll need a test Chef Automate server. https://learn.chef.io/modules/try-chef-automate#/setup is one way of setting one up locally.
+
+### Getting an API token
+
+After starting Automate, registering and starting a trial license, you will need to setup an API token. You will use this with `login`, which is required for most other commands.
+
+ 1. Click the "Settings" tab. A new set of menus appears on the left.
+ 2. Choose "API Tokens" from the left menu.
+ 3. Create a new API token. Name and ID don't really matter. After creation, select the new token, click the three dots ("extended menu") and choose "Copy Token".
+
+## Implementation Notes
+
+### Code Location
+
+#### as a CLI command plugin
+
+The codebase is located in `inspec/lib/plugins/inspec-compliance/`. It is a minimally converted v2 plugin, meaning that it works fine as a plugin but little was done to clean it up.
+
+#### Legacy external require location
+
+It appears that something - as in some other software package - is looking to require certain libs from paths like `inspec/lib/bundles/inspec-compliance`.  There are several stub files at that location. TODO - determine what software is doing that and either discourage such behavior or at least point them at the right location.
```

Comment:
I spy a "TODO" in this paragraph! :)

Reply:
Thanks, I tracked it down and revised accordingly.